### PR TITLE
Fix crash in /gamemode command

### PR DESCRIPTION
**Pull request by lleyton**
Currently, if a player issues /gamemode with an invalid game mode which is not a number (Ex: "/gamemode survival") the server crashes. This PR modifies a check to validate that the inputted game mode is a number from 0-3.

### CODE REVIEW

**Comment by rom1504** [comment]
Don't you think it would make more sense to handle /gamemode survival ?

**Comment by lleyton** [comment]
Yeah, that makes sense.

File: src/lib/plugins/players.js [hunk 1]
```diff
@@ -17,45 +17,46 @@ module.exports.server = function (serv, { version }) {
     base: 'gamemode',
     aliases: ['gm'],
     info: 'to change game mode',
-    usage: '/gamemode <0-3> [player]',
+    usage: '/gamemode <mode> [player]',
```
Comment (rom1504) [inline src/lib/plugins/players.js:20]: seems you now removed support completely for 0-3.
Vanilla handles both 0-3 and survival right?
maybe we could do the same?

Comment (lleyton) [reply src/lib/plugins/players.js:20]: Since 1.13, vanilla stopped handling numeric game mode IDs.

Comment (SiebeDW) [reply src/lib/plugins/players.js:20]: what about supporting them both?

**Pull request merged**